src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageMetadataControl.xaml

```diff
@@ -190,7 +190,7 @@ Background="{DynamicResource {x:Static nuget:Brushes.DetailPaneBackground}}"
         AutomationProperties.Name="{Binding Text}"
         Text="{x:Static nuget:Resources.Label_License}"/>
 
-      <ItemsControl Grid.Row="3" Grid.Column="1" ItemsSource="{Binding LicenseLinks}">
+      <ItemsControl Grid.Row="3" Grid.Column="1" ItemsSource="{Binding LicenseLinks}" IsTabStop="False">
```

Comment:
What does this do? Can the links still be accessed via tabbing?

Reply:
The link by itself forces a tab stop.  In this change, I'm removing a tab stop for the ItemsControl and the remaining tab stop would always be in the link.